source/slang/slang-diagnostic-defs.h

```diff
@@ -2551,6 +2551,8 @@ DIAGNOSTIC(
     attemptToQuerySizeOfUnsizedArray,
     "cannot obtain the size of an unsized array.")
 
+DIAGNOSTIC(56003, Error, useOfUninitializedResourceType, "use of uninitialized opaque type '$0'.")
```

Comment:
Need to update the diagnostic code name as well.

Reply:
a "type" cannot be "initialized", so here it should be "opaque handle", not "opaque type".